public/pages/CreatePolicy/containers/CreatePolicy/CreatePolicy.tsx

```diff
@@ -285,13 +288,7 @@ export class CreatePolicy extends Component<CreatePolicyProps, CreatePolicyState
         {this.renderEditCallOut()}
```

Comment:
Should the title on line 278 be EuiText size="s"

Reply:
No since this is old UI & it is main heading